src/util/renderers/renderer.builder.ts

```diff
@@ -0,0 +1,69 @@
+import MediaWiki from './../../MediaWiki.js'
+import { Renderer } from './abstract.renderer.js'
+import { VisualEditorRenderer } from './visual-editor.renderer.js'
+import { WikimediaDesktopRenderer } from './wikimedia-desktop.renderer.js'
+import { RendererBuilderOptions } from './../saveArticles.js'
```

Comment:
Here I believe `RendererBuilderOptions` should be defined here (at least I don't understand why this done that way).

Reply:
No feedback here?

Reply:
I decided to move it to the `src/util/renderers/abstract.renderer.ts`